docs/release-notes.txt

```diff
@@ -8,6 +8,52 @@
  Version 0.14
 **************
 
+Version 0.14.6
+==============
+
+**Release Date:** April 1, 2021
+
+**New Features**
+
+-  API: Add a new endpoint to delete experiments. This endpoint is only
+   enabled for admin users and deletes all resources associated with an
+   experiment. This includes checkpoint storage, Tensorboards, trial
+   logs from all backends and metadata such as history and metrics,
+   stored in PostgreSQL.
+
+-  API: Add a new endpoint to fetch aggregated historical resource
+   allocation information.
+
+-  CLI: Add new commands ``det resources raw`` and ``det resources
+   aggregated`` to access resource allocation information.
+
+-  PyTorch Lightning: Add an adapter to support ``LightningModule`` from
+   PyTorch Lightning in the PyTorchTrial API. :ref:`Read more about this
+   in the documentation here <lightning-adapter>`.
+
+-  WebUI: Experiment visualizations attempt to pick the most relevant
```

Comment:
blocking: More specifically, "hyperparameter visualizations" (i.e. not the main experimental details page). we should probably also mention how we calculate HP importance (very briefly, maybe just a link) -- @katport do you have a good link for the HP importance? also i'd move this to improvements.

also, let's use "hyperparameter" for HP everywhere.

Reply:
Do we want to add this for this release? It's not currently "on".

Reply:
Maybe something like: "The Hyperparameter visualizations will display the most relevant hyperparameters if the hyperparameter importance is available"?

Reply:
ah right. okay in that case maybe we leave this out until we're ready to turn it on?

Reply:
"available" is a bit confusing since it's probably not clear to a user when it's "available". let's just omit until we're ready to turn this on by default

Reply:
Omit what exactly?  The whole blurb?

Reply:
yes